netlogo-gui/resources/i18n/GUI_Strings_en.properties

```diff
@@ -573,11 +574,13 @@ the experiments'' "var" values will be:<br>\
 sequential order: 1, 1, 2, 2, 3, 3<br>\
 alternating order: 1, 2, 3, 1, 2, 3
 
-tools.behaviorSpace.metrics = Measure runs using these reporters:
+tools.behaviorSpace.metrics = Measure runs using these reporters as metrics:
 # this property has an HTML break in it.
 tools.behaviorSpace.metrics.info = one reporter per line; you may not split a reporter<br>across multiple lines
-tools.behaviorSpace.runMetricsEveryStep = Measure runs at every step
-tools.behaviorSpace.runMetricsEveryStep.info = if unchecked, runs are measured only when they are over
+tools.behaviorSpace.runMetricsEveryStep = Run metrics every step
```

Comment:
not actually for this item, but about: "I thought we'd discussed in the devel meeting making the checkbox/reporter for metric measurement collapsable, but I don't see that implemented here." We did discuss that at the meeting, the key point being to conserve GUI real estate. Isaac made a change so that the reporter would be on one line, which I think does the job. At any rate I think we should wait until after other GUI changes are made for other enhancements before deciding on what is best.